### PR TITLE
Made hydrogen peroxide function as disinfectant

### PR DESCRIPTION
**Pull request by Skidborg**
Moved diluted hydrogen peroxide from chemicals to medical comestibles, allowed it to be used to disinfect wounds (as the description claimed was possible), and be used to craft bandages and first aid kits as the disinfecting component.

### CODE REVIEW

**Comment by Coolthulhu** [comment]
No objections to bandage crafting, but apparently using it to disinfect wounds is a bad idea IRL.

File: data/json/items/comestibles.json [hunk 1]
```diff
@@ -14410,6 +14410,36 @@
         "fun" : 0
     },
     {
+        "type" : "COMESTIBLE",
+        "id" : "chem_hydrogen_peroxide",
+        "name" : "hydrogen peroxide",
+		"name_plural" : "hydrogen peroxide",
```
Comment (Coolthulhu) [inline data/json/items/comestibles.json]: Don't use tabs, use 4 spaces instead

File: data/json/recipes/recipe_others.json [hunk 2]
```diff
@@ -3109,7 +3109,8 @@
       [ "brandy", 7 ],
       [ "disinfectant", 1 ],
       [ "saline", 5 ],
-      [ "thyme_oil", 1 ]
+      [ "thyme_oil", 1 ],
+	  [ "chem_hydrogen_peroxide", 1 ]
```
Comment (Coolthulhu) [inline data/json/recipes/recipe_others.json]: Same here, also tab where 4 spaces should be

**Comment by drbig** [comment]
@Coolthulhu we were discussing on IRC the use of H2O2 as a disinfectant. It's used as such, and it works.

**Comment by drbig** [comment]
For completeness quoting out of the first paragraph on the wiki: "finds use as a strong oxidizer, bleaching agent and disinfectant".

**Comment by Rivet-the-Zombie** [comment]
>It's used as such, and it works.

It works great. The problem most Americans have with it is that it stings and can make the wound heal a little bit slower, but those are hardly issues when you're dealing with a life and death scenario like a zombie bite.

+1 for this PR!

**Comment by drbig** [comment]
> that it stings

I suggested it adding some pain (that's from my memories). Opinions were rather divided on that.

**Comment by Rivet-the-Zombie** [comment]
I wouldn't sweat it. The amount of pain is pretty inconsequential.

**Comment by drbig** [comment]
Agreed.

**Comment by i2amroy** [comment]
Actually I've looked into this quite a lot, and wounds treated with hydrogen peroxide, as well as having a longer heal time actually have a *higher* chance of becoming infected (like, several times higher). And unless you somehow manage to totally kill everything in the wound (which it's also bad at doing, it misses several key types of bacteria, thus why they no longer use it in hospitals), all you are going to do is buy yourself a very short respite from the infection, shortly followed by it reemerging at a much greater strength than before.

It works fine as a bandage preparation thing, but even in most serious scenarios it's not something doctors recommend that you do, and the belief that it is is basically pure momentum back from the days where we didn't know better. At best I'd vote for it's use as adding a separate effect. While that effect is present it blocks the infection effect, and then when it wears off you roll a die. If you get the lucky 1 in 10 roll it heals the infection (with an even lower chance for more advanced infections), else it just makes the infection even worse than it was before. More of a "I really need to buy myself another day to find an actual cure for this infection" delaying tactic than a cure.

**Comment by drbig** [comment]
So it's like "better than nothing now, not something you'd apply over a missing limb and declare you're fine". Sounds very plausible. I'm still for this getting in, we can add the "nice" roll death (infection rather?) later?

**Comment by Skidborg** [comment]
We do need much wider control of healing effects in jsons in general.  Even if it does work as a disinfectant, it shouldn't be directly healing wounds, as it is forced to do with stock disinfectant effects.

**Comment by i2amroy** [comment]
I can probably take a look into something like that sometime either tonight or tomorrow now that I've managed to get my development build set back up again. The end result could probably look like something quite similar to the current explosion usage info, just with more healy directed effects. :P

**Comment by drbig** [comment]
Factual update (via a retired hospital OSH compliance officer): hydrogen peroxide is fine to apply topically on *very small* wounds (rather: scratches). On anything nastier can mean pain up to losing consciousness and definitely tissue necrosis. Hospital use is mostly for sample preparation (didn't bother asking details).

Could be interesting to implement, given it'd have "Apply only to scratches." in description.

**Pull request merged**